go.mod

```diff
@@ -33,11 +33,11 @@ replace (
 
```

Comment:
> Once this PR is merged, I will tag the resulting latest commit as v0.2.4 (to be used only in the v2.6.3-patch1 release).

Are we going to release this tagged as 0.2.4, or as 2.6.3-patch1

Reply:
I had checked before opening the PR with @aiyengar2. We'd prefer not to disturb the `v0.2.4` series of chart RCs if possible.
Assuming we can have a chart with `2.6.3-patch1` as the version, we'd choose to go that route?

Reply:
I think that makes sense

Reply:
@maxsokolovsky @cbron We should not tag this release of rancher-webhook as `2.6.3-patch1`:

Anyone that updates to Rancher v2.6.3-patch1 would then get rancher-webhook `v2.6.3-patch1`. If they then tried to update to Rancher v2.6.4 when it is released, then the logic in Rancher would keep rancher-webhook version `v2.6.3-patch1` because this version is greater than the `MIN_WEBHOOK_VERSION` in Rancher v2.6.4 (which will be `v0.2.4`).

The problem with this is that rancher-webhook `v0.2.4` will have important updates in it related to CAPI that `v2.6.3-patch1` will not have. This is likely to break Rancher.

Reply:
Sorry we got mixed up on versions, we are going to tag this as 0.2.3-patch1. So you have: 
2.6.3: webhook 0.2.3
2.6.3-patch1: webhook 0.2.3-patch1
2.6.4: webhook 0.2.4
Going that route we should be fine, as 0.2.4 is greater than 0.2.3-patch1.